src/components/Discounts.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import DiscountCard from "./DiscountCard";
+import { View, Text } from "react-native";
+import { GET_DISCOUNTS } from "@app/queries/discounts";
+import { useQuery } from "@apollo/react-hooks";
+
+interface DiscountProps {
+  id: number;
+  content: DiscountContent;
+}
+
+interface DiscountContent {
+  title: string;
+  description: string;
+}
+
+const Discounts: React.FC = () => {
+  const { loading, error, data } = useQuery(GET_DISCOUNTS);
+  return (
+    <View>
+      <Text className="mb-8 text-2xl font-bold">Last Orders</Text>
+      {data &&
+        data?.DiscountItems.items.map((item: DiscountProps) => (
```

Comment:
why is `DiscountItems` capitalized?

Reply:
It comes from cms like this. It's a little bit weird since I cannot change. This is one of the things that I don't like about storyblok
<img width="236" alt="image" src="https://user-images.githubusercontent.com/59636753/224191137-4d4469f3-108d-4430-8773-dcba3f96954b.png">